libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -586,11 +603,120 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           ],
         },
         on: {
-          [DefaultEvents.APPROVE]: {
-            target: States.VINNUMALASTOFNUN_APPROVE_EDITS,
+          [DefaultEvents.EDIT]: { target: States.DRAFT },
```

Comment:
Please change this back to Approve and go to VinnumalaStofnunApproveEdits

Reply:
👍